test/functional/viml/function_spec.lua

```diff
@@ -27,3 +32,57 @@ describe('Up to MAX_FUNC_ARGS arguments are handled by', function()
     eq('Vim(call):E740: Too many arguments for function rpcnotify', ret)
   end)
 end)
+
+describe('setpos() function', function()
```

Comment:
would you mind putting this in a new file, `test/functional/eval/setpos_spec.lua` ? 

This file tests aspects of VimL functions. printf() and rpcnotify() were arbitrary examples used to test general function processing.